src/utils/messages.json

```diff
@@ -128,6 +129,8 @@
   "addprogram.field.label.allUsers": "Employees",
   "addprogram.input.date.format": "yyyy/MM/dd hh:mm a",
   "addprogram.input.date.mask": "___/__/__ __:__ _M",
+  "addprogram.validation.message.string":"This string should not be empty",
+  "addprogram.validation.message.array":"You should choose some item",
```

Comment:
I find this approach more friendly.
I recommend applying this in your application

![image](https://user-images.githubusercontent.com/75979545/144364347-9f4282d9-8568-4223-bdc0-db1e7f3e4aab.png)